pajbot/dispatch.py

```diff
@@ -402,7 +402,9 @@ def check_sub(bot, source, message, event, args):
 
     @staticmethod
     def remindme(bot, source, message, event, args):
+        syntax = "Syntax: !remindme TIME MESSAGE"
         if not message:
+            bot.say(f"{source}, No message provided! " + syntax)
```

Comment:
Why use the extra `syntax` variable? You could just put it in the string directly

```suggestion
        if not message:
            bot.say(f"{source}, No message provided! Syntax: !remindme TIME MESSAGE")
```

Reply:
There was another spot I was planning on putting this, but the message was never triggered, so I removed it.